### PR TITLE
Import sheets from 52, 55, 56, 59, 67, 73, s12, s26, s27, & sr08

### PR DESCRIPTION
**Pull request by ascii-dev**
## Description
Imports sheets 52, 55, 56, 59, 67, 73, s12, s26, & sr08 of the 2015 report file.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏾 @ascii-dev 

-------------------

Aren't we supposed to get the names & description from the `GMMP_2020_Queries.xlsm` file? For example

_ws_27,_ `title` in this PR is **Breakdown of News subjects photographed, by sex** and `description` is **News subjects photographed, by sex**. But looking at the queries sheet 👇🏾 , for 2020 `title` is supposed to be **News subjects photographed, by sex** and `description` **B.down of  News subjects photographed, by sex**
Screenshot from 2021-01-29 08-35-32

**Comment by ascii-dev** [comment]
> 👍🏾 @ascii-dev
> 
> Aren't we supposed to get the names & description from the `GMMP_2020_Queries.xlsm` file? For example
> 
> _ws_27,_ `title` in this PR is **Breakdown of News subjects photographed, by sex** and `description` is **News subjects photographed, by sex**. But looking at the queries sheet 👇🏾 , for 2020 `title` is supposed to be **News subjects photographed, by sex** and `description` **B.down of News subjects photographed, by sex**
> Screenshot from 2021-01-29 08-35-32

Oh... Shoot... I assumed the title and description we had in `WS_INFO` corresponded with what's in the queries sheets, + yesterday was my actual first time seeing the queries sheet. I'll look into descriptions that have changed and update them. @kilemensi

File: reports/report_details.py [hunk 1]
```diff
@@ -380,6 +380,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_27': {
+        '2015': {
+            'name': '27',
+            'title': 'Breakdown of News subjects photographed, by sex',
+            'desc': 'News subjects photographed, by sex',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'title': 'News subjects photographed, by sex',
            'desc': 'Breakdown of News subjects photographed, by sex',
```

File: reports/report_details.py [hunk 2]
```diff
@@ -414,6 +421,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_31': {
+        '2015': {
+            'name': '31',
+            'title': 'Breakdown of Reporters, by sex on different topics - Detail',
+            'desc': 'Rporters, by sex on different topics - Detail',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'title': 'Reporters, by sex on different topics - Detail',
            'desc': 'Breakdown of Reporters, by sex on different topics - Detail',
```

File: reports/report_details.py [hunk 3]
```diff
@@ -599,6 +683,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_55': {
+        '2015': {
+            'name': '55',
+            'title': 'Internet - Occupation of Female news subjects',
+            'desc': 'B.down Internet - Occupation and function by sex of News Subjects',
+            'reports': ['global'],
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'reports': ['global', 'region', 'country'],
```

**Review by kilemensi — COMMENTED** [review]
👍🏾 @ascii-dev ... Lots of queries where title and description have been criss-crossed. Please double check.

QQ: All these sXX, srXX, are no longer there in the queries sheet, correct?

**Comment by ascii-dev** [comment]
> 👍🏾 @ascii-dev ... Lots of queries where title and description have been criss-crossed. Please double check.
> 
> QQ: All these sXX, srXX, are no longer there in the queries sheet, correct?

Yes, but there's also 100 - 111 which I'm about to confirm from `Jean` what they mean. But since the sXX and srXX are no longer in the queries sheet, does that mean I don't have to import them? @kilemensi

**Comment by kilemensi** [comment]
bunch of conflicts @ascii-dev

**Comment by ascii-dev** [comment]
> bunch of conflicts @ascii-dev 

Currently working on fixing the conflicts. @kilemensi

**Review by kilemensi — APPROVED** [review]
🚀 @ascii-dev 


QQ: When importing sheet like `55`, are we supposed to also import **N** column?

**Comment by ascii-dev** [comment]
Yes, according to what I understand from my discussions with 'Jean', it helps keep the percentages in perspective.

**Comment by ascii-dev** [comment]
@kilemensi 👆

**Comment by kilemensi** [comment]
> Yes, according to what I understand from my discussions with 'Jean', it helps keep the percentages in perspective.

Cool, I'm looking at the `historical.json` for sheet 55 and I don't see it @ascii-dev

**Comment by ascii-dev** [comment]
@kilemensi https://github.com/CodeForAfrica/gmmp/pull/120/commits/f1753cf444686ff03015d9866850b6ff50b7663d#diff-e0b8644d16d155669d4a77d2a2863f07ae2ce51bbacfbf107d4d11bfb3f91773R7524

**Comment by kilemensi** [comment]
> @kilemensi [f1753cf#diff-e0b8644d16d155669d4a77d2a2863f07ae2ce51bbacfbf107d4d11bfb3f91773R7524](https://github.com/CodeForAfrica/gmmp/commit/f1753cf444686ff03015d9866850b6ff50b7663d#diff-e0b8644d16d155669d4a77d2a2863f07ae2ce51bbacfbf107d4d11bfb3f91773R7524)

For some reason this is what I get locally:
Screenshot from 2021-02-02 17-23-14

Will try running it again after the call @ascii-dev ... but lets move forward with the PR since it seems to work.

**Comment by ascii-dev** [comment]
You need to update the header to also include N. @kilemensi 
image

**Comment by kilemensi** [comment]
> You need to update the header to also include N. @kilemensi
> image

Ahh... Is the one in gFolder fixed @ascii-dev ?

**Comment by ascii-dev** [comment]
I'm not sure it's fixed yet. I'll liaise with Jean to fix the one in the folder as soon as I'm through with everything so everyone has the updated version. @kilemensi

**Comment by kilemensi** [comment]
> I'm not sure it's fixed yet. I'll liaise with Jean to fix the one in the folder as soon as I'm through with everything so everyone has the updated version. @kilemensi

Cool, cool @ascii-dev... just checked and it's not fixed so please don't forget.
Screenshot from 2021-02-02 17-42-09

**Pull request merged**